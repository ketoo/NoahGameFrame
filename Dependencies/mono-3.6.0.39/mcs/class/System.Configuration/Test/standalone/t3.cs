using System;
using System.Collections.Generic;
using System.Collections.Specialized;
using System.Text;
using System.Configuration;
using System.Web;

class T1
{
	static void Main(string[] args)
	{
		try
		{
			NameValueCollection AppSettings = ConfigurationManager.AppSettings;
			Configuration config = ConfigurationManager.OpenExeConfiguration (ConfigurationUserLevel.None);

			AppSettingsSection appsettings = config.AppSettings;

			Console.WriteLine("IsMachineLevel: {0}", config.EvaluationContext.IsMachineLevel);

			Console.WriteLine ("appsettings.Settings.AllKeys.Count = {0}", appsettings.Settings.AllKeys.Length);

			foreach (string key in appsettings.Settings.AllKeys) {
				Console.WriteLine ("settings[{0}] = {1}", appsettings.Settings[key].Key, appsettings.Settings[key].Value);
			}

			foreach (string key in AppSettings.AllKeys) {
				Console.WriteLine ("AppSettings[{0}] = {1}", key, AppSettings[key]);
			}
		}
		catch (Exception e)
		{
			// Error.
			Console.WriteLine(e.ToString());
		}
	}
}
