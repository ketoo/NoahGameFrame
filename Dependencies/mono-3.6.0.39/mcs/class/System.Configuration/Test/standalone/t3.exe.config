<configuration>

<appSettings>
	<add key="hithere" value="Hello World!"/>
	<add key="second"  value="Welcome World!"/>
	<add key="hithere" value="Goodbye World!"/>
</appSettings>

</configuration>
