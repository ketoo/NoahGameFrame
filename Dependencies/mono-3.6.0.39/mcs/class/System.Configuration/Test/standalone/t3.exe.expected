IsMachineLevel: False
appsettings.Settings.AllKeys.Count = 2
settings[second] = Welcome World!
settings[hithere] = Goodbye World!
AppSettings[second] = Welcome World!
AppSettings[hithere] = Goodbye World!
